cpp/src/sqls/sqls_rtti_comp.h

```diff
@@ -927,86 +921,58 @@ size_t multi_col_group_by_avg_sort(size_t         nrows,
 
 
 //###########################################################################
-//#                          Multi-column ORDER-BY:                         #
+//#                          Multi-column Sort:                             #

```

Comment:
Please update the documentation to fit the Doxygen template here: https://github.com/rapidsai/cudf/blob/3c7bea328533a78907ab25871d10095e9a838c87/libgdf/src/example_documentation.cpp